src/overlay/widgets/Settings.cpp

```diff
@@ -54,7 +54,7 @@ void Settings::OnUpdate()
         m_madeChanges = false;
         if (ImGui::CollapsingHeader("Patches", ImGuiTreeNodeFlags_DefaultOpen))
         {
-            ImGui::TreePush();
+            ImGui::TreePush((void*)nullptr);
```

Comment:
I would just input the actual table constant this is refering to, which is the one in the if

i believe it accepts string, and if it doesnt, just cast that to void*

also, cast using static_cast please, not C style cast.

applies to similar cases

Reply:
reasoning - anything using nullptr would get treated the same way, this could break the UI (unsure if it really does atm, but it makes this share the ID of the push with other windows/childs/whatev these are in the end...)

it is bug prone if accidentally closed the wrong way